src/Upload/Upload.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {UploadButton} from '..';
+import AntdUpload from 'antd/lib/upload/Upload';
+import message from 'antd/lib/message';
+
+import './Upload.scss';
+
+const getListType = (type) => {
+    if (typeof type === 'object') {
+        if (type.image) {
+            return 'picture';
+        }
+
+    } else if (typeof type === 'string') {
+        if (type === 'image') {
+            return 'picture';
+        }
+    }
+};
+
+const getFileTypes = (type) => {
```

Comment:
wenn es kein object ist wird nichts zurückgegeben

Reply:
ja, dann wird die abfrage in der validate methode nicht ausgeführt. brauch sie dann auch nicht.. könnte natürlich auch explizit false zurückgeben aber ich dachte ich spare mir code

Reply:
es wäre sauberer etwas zurückzugeben

Reply:
done